packages/terra-list/src/List.jsx

```diff
@@ -34,18 +34,6 @@ const propTypes = {
     * compared to `aria-describedby`, this prop should only be used in cases where
     * `aria-describedby` is not sufficient.
     */
-  ariaDescription: PropTypes.string,
-  /**
-   * References a visible or visually hidden text element
-   * on the page that conveys information about the list
-   * and how to interact with it. Maps to
-   * [aria-details](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-details).
-   * Can be used alongside `ariaDescribedBy` or `ariaDescription`.
-   * `aria-details` is not supported in all screen reader/browser
-   * combinations. Consumers should test for different screen
-   * reader/browser combinations to ensure that instructions
-   * are consistently read.
-   */
```

Comment:
you need to remove comments added for ariaDescription not the comments added for ariaDetails

Reply:
Updated.

Reply:
Remove description of `ariaDescription` and restore description of `ariaDetails`. 
`ariaDetails` prop needs to be uncommented.

Reply:
Updated..